README.md

```diff
@@ -12,7 +12,7 @@ maven
 <dependency>
     <groupId>com.redhat.darcy</groupId>
     <artifactId>darcy-webdriver</artifactId>
-    <version>0.2.2-SNAPSHOT</version>
+    <version>0.3.0-SNAPSHOT</version>
```

Comment:
Let's bump to 0.3.1 snapshot. (Your original revision also bumped the patch version, so let's be consistent since I think that's not a bad idea in order to not surprise people too much)